.github/workflows/create-release.yml

```diff
@@ -1,279 +1,514 @@
+          name: current-nodejs-18-run-image
+          path: ./build-nodejs-18/run.oci
+
+      - name: Get nodejs 18 run image hash code
+        run: |
+          mkdir -p hash-codes/build-nodejs-18
+
+          ubi_images_need_update=($(echo '${{ needs.poll_ubi_images.outputs.ubi_images_need_update }}' | jq -r '.[]'))
+          ubi_image_previous_hash_codes=($(echo '${{ needs.poll_ubi_images.outputs.ubi_image_previous_hash_codes }}' | jq -r '.[]'))
+
+          current_run_nodejs_18_sha256_image_hash_code=""
+          if echo "${ubi_images_need_update[@]}" | grep -qw "node-18-run"; then
+            docker pull registry.access.redhat.com/ubi8/nodejs-18-minimal
+            current_run_nodejs_18_sha256_image_hash_code=$(docker image inspect registry.access.redhat.com/ubi8/nodejs-18-minimal --format '{{json .RepoDigests}}' | jq -r '.[0]')
+          else
+            for ((i = 0; i < ${#ubi_image_previous_hash_codes[@]}; i += 2)); do
+              if [ "${ubi_image_previous_hash_codes[i]}" == "node-18-run" ]; then
+                current_run_nodejs_18_sha256_image_hash_code=${ubi_image_previous_hash_codes[i + 1]}
+              fi
+            done
+          fi
+
+          echo $current_run_nodejs_18_sha256_image_hash_code > hash-codes/build-nodejs-18/run.oci.sha256
+
+      - name: Upload nodejs 18 run image hash code
+        uses: actions/upload-artifact@v3
+        with:
+          name: current-nodejs-18-run-image-hash-code
+          path: hash-codes/build-nodejs-18/run.oci.sha256
 
   test:
     name: Acceptance Test
-    needs: [ create_stack ]
+    needs: [create_stack]
     runs-on: ubuntu-22.04
     steps:
-    - name: Setup Go
-      uses: actions/setup-go@v3
-      with:
-        go-version: 1.20.x
-
-    - name: Checkout
-      uses: actions/checkout@v3
-
-    - name: Create OCI artifacts destination directory
-      run: |
-        mkdir -p build
-
-    - name: Download Build Image
-      uses: actions/download-artifact@v3
-      with:
-        name: current-build-image
-        path: build
-
-    - name: Download Run Image
-      uses: actions/download-artifact@v3
-      with:
-        name: current-run-image
-        path: build
-
-    - name: Download nodejs-16 Run Image
-      uses: actions/download-artifact@v3
-      with:
-        name: current-nodejs-16-run-image
-        path: build-nodejs-16
-
-    - name: Download nodejs-18 Run Image
-      uses: actions/download-artifact@v3
-      with:
-        name: current-nodejs-18-run-image
-        path: build-nodejs-18
-
-    #- name: Run Acceptance Tests
-    #run: ./scripts/test.sh
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.20.x
+
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Create OCI artifacts destination directory
+        run: |
+
+      - name: Run Acceptance Tests
+        # run: ./scripts/test.sh
+        run: |
+          echo "ommit for now, till acceptance tests pass."
```

Comment:
We should leave the downloads in place as they are needed for the acceptance tests.  That should be the next thing we work on getting running.

Reply:
Agree

Reply:
I also commented out a few codes on acceptance tests in order to pass on this commit https://github.com/paketo-community/ubi-base-stack/pull/7/commits/d09ff0d436524c51b31b1dd3caba3ba80834790e . 
So instead of having this 
```
 run: |
          echo "ommit for now, till acceptance tests pass."
```

to have this `run: scripts/test.sh`

Reply:
I think running the tests with that part commented out is better, but a bit confused since I think that test used to run pass?

Reply:
@pacostas and I'm not sure why the file /etc/os-release would not be present?  If I run `docker run -it paketocommunity/run-nodejs-18-ubi-base /bin/bash` I can see that the file is there?

Reply:
The tests were failing, but now they dont. Probably it was pulling an older image where the `/etc/os-release` wasnt present?
Reverted the changes on this commit https://github.com/paketo-community/ubi-base-stack/pull/7/commits/7d17a2efe649c96eeb2494804996f9e57d821fc5